.evergreen/config.yml

```diff
@@ -1431,14 +1439,6 @@ tasks:
       - func: run-aws-auth-test-with-aws-EC2-credentials
       - func: run-aws-ECS-auth-test
 
-  - name: go1.10-build-cse
```

Comment:
This task isn't needed anymore because the `build` target now respects the tags passed through the `BUILD_TAGS` env var, so it will compile the `cse` files.